core-web/src/main/java/org/silverpeas/core/web/authentication/AuthenticationServlet.java

```diff
@@ -125,10 +140,10 @@ public void doPost(HttpServletRequest servletRequest, HttpServletResponse servle
     userCanTryAgainToLoginVerifier.clearSession(request);
 
     if (authService.isInError(authenticationKey)) {
-      processError(authenticationKey, request, servletResponse, authenticationParameters,
+      processError(authenticationKey, request, response, authenticationParameters,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a "try/catch" block for "processError". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1989)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a "try/catch" block for "processError". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1989)